src/locales/en.json

```
@@ -127,17 +128,32 @@
     "charts": {
         "common": {
             "loading": "Loading data…",
+            "from": "",
+            "to": "-",
+            "zoom": "Zoom",
+            "resetZoom": "Reset zoom",
+            "resetZoomTitle": "Reset zoom level 1:1",
+            "1m": "1m",
+            "2m": "2m",
+            "3m": "3m",
```

Bi bilo bolj smiselno namesto treh stringov imeti enega z več množinslimi oblikami? Npr:
```suggestion
            "x_months": "{{ count }} month",
            "x_months_plural": "{{ count }} months",
```
Lahko tudi kasneje.

---

Se strinjam, sem popravila.